lib/workers/package/index.js

```diff
@@ -16,6 +17,9 @@ async function renovatePackage(config) {
     logger.debug('package is disabled');
     return [];
   }
+  if (config.depType === 'docker') {
+    return renovateDockerImage(config);
```

Comment:
The `renovatePackage` function seems ideal for a re-factor since there's a high-level call here to `renovateDockerImage` yet, there's low-level `npm` logic in the proceeding line.